types/human-date/human-date-tests.ts

```diff
@@ -0,0 +1,6 @@
+import hdate = require('human-date');
+
+hdate.prettyPrint(new Date());
+hdate.relativeTime('12-11-1998');
```

Comment:
Please update to include optional parameters, as per documentation:
```suggestion
hdate.relativeTime('12-11-1998');
hdate.relativeTime(4, {futureSuffix: "in the future"});
hdate.relativeTime(new Date("8-16-1987"), {returnObject: true});
```